### PR TITLE
Suit release 2022 6

### PR DESCRIPTION
**Pull request by robyww**
SUIT updates for firefly release `2022.3`

test on `data-int`

### CODE REVIEW

File: src/suit/js/SUIT.js [hunk 1]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                    ],
+                },
+            ],
+            obsCoreCalibrationLevel: {
+                tooltip: 'Calibration ',
+                helptext: '1 is raw data; 2 is PVIs; 3 selects difference images and coadds',
+                level: {
+                    1: {title: 'For Rubin: Raw Data'},
+                    2: {title: 'For Rubin: PVIs'},
+                    3: {title: 'For Rubin: Difference Images and Coadds'},
```
Comment (gpdf) [inline src/suit/js/SUIT.js:150]: Let's change this to 'For Rubin: Coadds and Difference Images'

File: src/suit/js/SUIT.js [hunk 2]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                        {label: 'r', value : '622', title: '622nm central value'},
+                        {label: 'i', value : '755', title: '755nm central value'},
+                        {label: 'z', value : '869', title: '869nm central value'},
+                        {label: 'y', value : '971', title: '971nm central value'},
+                    ],
+                },
+            ],
+            obsCoreCalibrationLevel: {
+                tooltip: 'Calibration ',
+                helptext: '1 is raw data; 2 is PVIs; 3 selects difference images and coadds',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:146]: '1 is raw data; 2 is PVIs; 3 includes coadds and difference images'

File: src/suit/js/SUIT.js [hunk 3]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                        {label: 'g', value : '483', title: '483nm central value'},
+                        {label: 'r', value : '622', title: '622nm central value'},
+                        {label: 'i', value : '755', title: '755nm central value'},
+                        {label: 'z', value : '869', title: '869nm central value'},
+                        {label: 'y', value : '971', title: '971nm central value'},
+                    ],
+                },
+            ],
+            obsCoreCalibrationLevel: {
+                tooltip: 'Calibration ',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:145]: 'Calibration Level'

File: src/suit/js/SUIT.js [hunk 4]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                helptext: '1 is raw data; 2 is PVIs; 3 selects difference images and coadds',
+                level: {
+                    1: {title: 'For Rubin: Raw Data'},
+                    2: {title: 'For Rubin: PVIs'},
+                    3: {title: 'For Rubin: Difference Images and Coadds'},
+                },
+            },
+            obsCoreInstrumentName: {
+                tooltip: 'LSSTCam, LSSTComCam',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:154]: 'LSSTCam, LSSTCam-imSim'

File: src/suit/js/SUIT.js [hunk 5]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                placeholder: 'e.g. LSSTCam',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:155]: 'e.g., LSSTCam-imSim'

Will it be possible to further override just this one thing in deployment configurations?  Or does the whole override from here have to get replaced then?

File: src/suit/js/SUIT.js [hunk 6]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+            },
+            obsCoreSubType: {
+                tooltip: 'Butler Dataset Type (raw, calexp, etc..)',
+                placeholder: 'e.g. calexp',
+                helptext: 'Butler Repo Dataset type',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:160]: '"lsst." + Butler Dataset Type'

File: src/suit/js/SUIT.js [hunk 7]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                    3: {title: 'For Rubin: Difference Images and Coadds'},
+                },
+            },
+            obsCoreInstrumentName: {
+                tooltip: 'LSSTCam, LSSTComCam',
+                placeholder: 'e.g. LSSTCam',
+            },
+            obsCoreSubType: {
+                tooltip: 'Butler Dataset Type (raw, calexp, etc..)',
+                placeholder: 'e.g. calexp',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:159]: 'e.g., lsst.deepCoadd_calexp'

File: src/suit/js/SUIT.js [hunk 8]
```diff
@@ -121,6 +126,41 @@ let options = {
...
+                    2: {title: 'For Rubin: PVIs'},
+                    3: {title: 'For Rubin: Difference Images and Coadds'},
+                },
+            },
+            obsCoreInstrumentName: {
+                tooltip: 'LSSTCam, LSSTComCam',
+                placeholder: 'e.g. LSSTCam',
+            },
+            obsCoreSubType: {
+                tooltip: 'Butler Dataset Type (raw, calexp, etc..)',
```
Comment (gpdf) [inline src/suit/js/SUIT.js:158]: 'Specific type of image or other dataset'

**Review by gpdf — CHANGES_REQUESTED** [review]
Getting close, just some fine-tuning vs. what Brian originally had.

File: src/suit/js/SUIT.js (hunk 5 above)
Comment (robyww) [reply src/suit/js/SUIT.js:155]: we attempt to substitute the config object like that. I think it would work but we would have to test it.

**Pull request merged**